src/pybind/mgr/progress/module.py

```diff
@@ -142,7 +142,7 @@ def __init__(self, message, refs, which_pgs, evacuate_osds):
     def evacuating_osds(self):
         return self. _evacuate_osds
 
-    def pg_update(self, pg_dump, log):
+    def pg_update(self, pg_dump, log,latest_osd_map_epoch):
```

Comment:
instead of latest_osd_map_epoch, we want something like ``start_epoch`` on the progress event. add a property there that gets the (current) osdmap when the event is created, and compare against that

Reply:
nit: space after ``,``